Priyanker/Assignment No 2/Project2.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent" >
+
+    <TextView
+        android:id="@+id/title"
+        android:layout_width="300dp"
+        android:layout_height="37dp"
+        android:text="@string/fcc"
+        android:textSize="30sp"
+        android:paddingLeft="50dp"
+        android:textAlignment="center"
+        android:textColor="@android:color/white"
+        android:background="#006203"
+        android:fontFamily="sans-serif"
+        android:layout_alignParentTop="true"/>
+    <ImageView
+        android:id="@+id/symbol"
+        android:layout_width="39.3dp"
+        android:layout_height="39.9dp"
+        android:layout_toRightOf="@id/title"
+        android:src="@drawable/fccsymbol"
+        android:layout_alignParentTop="true" />
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="37dp"
+        android:background="#006203"
+        android:layout_toRightOf="@id/symbol"
+        android:layout_alignParentTop="true"/>
+    <ImageView
+        android:id="@+id/mocpicture"
+        android:layout_width="match_parent"
+        android:layout_height="200dp"
+        android:layout_below="@id/title"
+        android:layout_alignParentLeft="true"
+        android:src="@drawable/mocpic"
+        android:scaleType="centerCrop"/>
+
+    <TextView
+        android:id="@+id/textView2"
+        android:layout_width="match_parent"
+        android:layout_height="30dp"
+        android:layout_below="@id/mocpicture"
+        android:background="@android:color/holo_orange_light"
+        android:paddingTop="2dp"
+        android:text="@string/moc"
+        android:textAlignment="center"
+        android:textColor="?attr/colorPrimaryDark"
+        android:textSize="20sp"
+        android:textStyle="bold" />
+    <LinearLayout
+        android:id="@+id/mocweb"
+        android:background="#4eb980"
+        android:layout_width="wrap_content"
```

Comment:
OH yes! typed a long text. So I added wrap_content. Didn't notice that. 
Will change it.